Vagrantfile

```diff
@@ -157,11 +158,21 @@ scissor = {
     :net_type => NETWORK_TYPE_DHCP,
     :show_gui => false
   },
+  CNIT_CORRELATOR_VM_NAME => {
+    :autostart => false,
+    :box => CENTOS_BOX_ID,
+    :cpus => 1,
+    :mac_address => "0800271F9D11",
+    :mem => 2048,
+    :net_auto_config => false,
+    :net_type => NETWORK_TYPE_DHCP,
+    :show_gui => false
+  },
```

Comment:
Add an entry in `docker/scissor-dnsmasq/etc/dhcp-hosts/static-ip-leases.conf` if you want your VM to have a fixed IP. Otherwise leave it as it is.